core/amber/src/main/scala/edu/uci/ics/texera/web/service/WorkflowJobService.scala

```diff
@@ -66,6 +66,15 @@ class WorkflowJobService(
       )
     }
     resultService.attachToJob(stateStore, workflowInfo, client)
+
```

Comment:
remove this, since now the insertion of a new execution happens in the workflow service